clients/python/marquez_client/client.py

```diff
@@ -348,6 +348,16 @@ def list_tags(self, limit=None, offset=None):
             }
         )
 
+    def create_tag(self, name, description=None):
+        payload = {
```

Comment:
_Minor:_ We may want to check that the `description` is not `None`, otherwise the `None` value would be set to the description in the payload?

Reply:
A `None` value will be transformed to `null` in the payload json. According to the `tag` schema the description may be null and the java api also allows a `null` value here. It explicitly uses an Optional as return value for getDescription.  
We could use `""` (empty string) as the default value but that would kind of break the contract in the java api. Not sure though if that is the intended behaviour of the tags api

Reply:
Sounds good. As long as `None` maps to a `null`, that's the intended behavior. My comment was more around whether we wanted to add the following check:

```
if description:
     payload['description'] = description
```